packages/ckeditor5-image/src/image/converters.js

```diff
@@ -107,9 +108,16 @@ export function srcsetAttributeConverter() {
 	}
 }
 
+/**
+ * Converter used to convert a given image attribute from the model to the view.
+ *
+ * @param {String} attributeKey The name of the attribute to convert.
+ * @returns {Function}
+ */
 export function modelToViewAttributeConverter( attributeKey ) {
 	return dispatcher => {
 		dispatcher.on( `attribute:${ attributeKey }:image`, converter );
+		dispatcher.on( `attribute:${ attributeKey }:imageInline`, converter );
```

Comment:
Those should be registered separately by `ImageBlock` and `ImageInline`.

Reply:
Done.